addons/hr_holidays/static/src/dashboard/time_off_card.js

```diff
@@ -41,6 +42,27 @@ export class TimeOffCardPopover extends Component {
             domain: [["id", "in", this.props.errorLeaves]],
         });
     }
+
+    async navigateInfo(state) {
+        this.actionService.doAction({
+            type: "ir.actions.act_window",
+            name: "My Time Off",
+            res_model: "hr.leave",
+            views: [
+                [false, "list"], 
+                [false, "form"],
+            ],
+            domain: [
+                ['user_id', '=', user.userId],
```

Comment:
thanks for your work :smiley: 

could be one getter for domain so that you can call single action service for diff purpose. :slightly_smiling_face: